### DISCUSSION TITLE
在baseHandlers里 在get和set的时候  使用Reflect和target[key]两种方式有何区别?

### DISCUSSION THREAD

**Discussion by atlasyoung** [cuiyiming1998/Vue3-mini#6] [Q&A] [answered]
在baseHandlers里 在get和set的时候  使用`Reflect`和`target[key]`两种方式有何区别?

**Comment by cuiyiming1998** [comment, marked as answer]
简单来讲 -> 如果访问的属性 是一个访问器属性 例如`get name() { return 2 }` 这时获取的值会出现错误

如果`get name() { return this.foo }`这种情况, 响应式关系是建立不起来的
